src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -113,6 +115,16 @@ private String initialRespondentDetailsMarkUp(CaseData caseData) {
         }
     }
 
+    private String initialAcasCertList(CaseData caseData) {
```

Comment:
It would be much nicer to have all the private methods in their own section under all the public method